CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
   precedence.
 - `singularity.conf` now accepts setting new options regarding namespaces:
   - `allow ipc ns` : disable the use of the `--ipc` flag.
+  - `allow user ns` : disable the use of any `--userns` or `--fakeroot` flags,
```

Comment:
> disable creation of user namespaces. This will prevent execution of containers with the `--userns` or `--fakeroot` flags, and unprivileged installations of SingularityCE.